hivemind/utils/crypto.py

```diff
@@ -0,0 +1,84 @@
+import re
```

Comment:
Question ( @borzunov , @mryab ): should we store crypto here or in hivemind/dht/crypto.py?

In other words, shall we need the same type of validator outside DHT?

Reply:
I don't mind moving it to `hivemind/dht/crypto.py` :)

Reply:
Done